### ISSUE TITLE
[NTI] False "All constants must be typed" warning on namespaced constant properties

### ISSUE THREAD

**Issue by Dominator008** [google/closure-compiler#1725]
I'm getting a 500 on [the debugger](https://closure-compiler-debugger.appspot.com) so I can't include a link.
```js
// hasOwnProperty.js
/** @const */ var ns = {};
/** @const */ ns.x = Object.prototype.hasOwnProperty;
```
Run
```sh
java -jar build/compiler.jar -W VERBOSE --new_type_inf hasOwnProperty.js 
```
and I get
```
hasOwnProperty.js:2: WARNING - All constants must be typed. The compiler could not infer the type of this constant. Please use an explicit type annotation.
/** @const */ ns.x = Object.prototype.hasOwnProperty;
```
`Object.prototype.hasOwnProperty` is part of the ES3 externs so it's already typed.

Real-world example:
https://github.com/google/incremental-dom/blob/70ad014d89d42818584f88886e6896e15535f806/src/util.js#L21

**Comment by Dominator008** [comment]
@dimvar FYI, this is the biggest false positive I got when checking [Incremental DOM](https://github.com/google/incremental-dom) with NTI. Once this is fixed I will switch them :)

**Comment by dimvar** [comment]
Say no more :) Upgrading.
